pkg/schema/tracker.go

```diff
@@ -92,6 +95,33 @@ func (tr *Tracker) GetTable(db, table string) (*model.TableInfo, error) {
 	return t.Meta(), nil
 }
 
+// GetCreateTable returns the `CREATE TABLE` statement of the table.
+func (tr *Tracker) GetCreateTable(ctx context.Context, db, table string) (string, error) {
+	name := dbutil.TableName(db, table)
+	// use `SHOW CREATE TABLE` now, another method maybe `executor.ConstructResultOfShowCreateTable`.
+	rs, err := tr.se.Execute(ctx, fmt.Sprintf("SHOW CREATE TABLE %s", name))
+	if err != nil {
+		return "", err
+	}
+	defer rs[0].Close()
+
+	req := rs[0].NewChunk()
+	err = rs[0].Next(ctx, req)
+	if err != nil {
+		return "", err
+	}
+	if req.NumRows() == 0 {
+		return "", nil // this should not happen.
+	}
+
+	row := req.GetRow(0)
+	str := row.GetString(1) // the first column is the table name.
```

Comment:
```suggestion
	str := row.GetString(1) // the first column is the table schema.
```

do you mean this?

Reply:
No, the first (`0`) is the name, the second(`1`) is the schema.